metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/controller/GenericEntitiesController.java

```diff
@@ -128,13 +128,21 @@ protected List<E> buildEntityList(
                             aspectNames.stream()
                                 .map(aspectName -> Map.entry(aspectName, 0L))
                                 .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue))))
-                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)));
+                .collect(
+                    Collectors.toMap(
+                        Map.Entry::getKey,
+                        Map.Entry::getValue,
+                        (a, b) -> {
+                          throw new IllegalStateException("Duplicate key");
```

Comment:
If it has the same value then it's likely okay, do we still want to throw an error for this case? Or can we put in a comment why the default map merger function is not appropriate?

Reply:
My understanding is that this is the default behavior of the map classes. Unfortunately the steaming interface doesn't allow deferring to the default behavior and forces me to re-implement the merge